tools/dockerfile/interoptest/grpc_interop_python/Dockerfile

```diff
@@ -73,6 +73,14 @@ RUN apt-get update && apt-get install -y \
     python3-all-dev \
     python-pip
 
+# Install Pyenv and associated Python versions
+RUN curl -L https://raw.githubusercontent.com/yyuu/pyenv-installer/master/bin/pyenv-installer | sh
+RUN pyenv update
+RUN pyenv install 2.7-dev
```

Comment:
No, it doesn't. The reasoning behind this was that pyenv will always fetch the latest dev versions as opposed to PPAs. Theoretically, there shouldn't be any breaking changes. In case there are, running two separate tests, one on the PPA and one on pyenv would warn us of future failures in advance. I agree that it is a bit of an overkill. We can just use pyenv for 3.5 and 3.6 installations only.